docs/design/numeric_literals.md

```diff
@@ -13,10 +13,8 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 ## Table of contents
```

Comment:
I don't think you should be changing this document. This document is talking about numeric literals, like `0`, `-7`, and `3.14`, and we want to keep this content. Numeric type literals are a different thing, and would be covered by a different document.

Reply:
I see. I will create new md document for numeric type literals.